processing/src/main/java/io/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -112,7 +133,13 @@ public void stop()
    */
   public boolean put(String lookupName, final LookupExtractorFactory lookupExtractorFactory)
   {
-    synchronized (lock) {
+    try {
+      startStopLock.readLock().lockInterruptibly();
+    }
+    catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
1. Is the error suppression intentional? 
2. Should this use a logger rather than printStackTrace?